src/atelier.cpp

```diff
@@ -83,3 +96,74 @@ void Atelier::printFile(const QString &fileName)
 void Atelier::percentagePrinted(){
     emit percentage(core->percentagePrinted());
 }
+
+void Atelier::writeTempFile(QString text)
+{
+    /*
+    A QTemporaryFile will always be opened in QIODevice::ReadWrite mode,
+    this allows easy access to the data in the file. This function will
+    return true upon success and will set the fileName() to the unique
+    filename used.
+    */
+    logFile->open();
+    logFile->seek(logFile->size());
+    logFile->write(text.toLatin1());
+    logFile->putChar('\n');
```

Comment:
BTW I was wrong, that convertion don't work. 
/home/lays/programs/atelier/interface/src/atelier.cpp:127: error: no matching function for call to ‘QChar::fromLatin1(QString&)’
     logFile->write(QChar::fromLatin1(text));

the write expect a ByteArray. I'm going to undo this. 
I already tried with QString::toLatin1, since is a convertion but don't work. (too many arguments), and from also don't work. So apparently toLatin1 is the best solution.